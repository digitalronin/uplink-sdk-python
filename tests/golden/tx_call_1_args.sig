r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 47461933023237928146702596291579784760845614502239423412114586705902891697117
