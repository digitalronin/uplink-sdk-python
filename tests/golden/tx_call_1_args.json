{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VInt", 
                        "contents": 1
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE00NzQ2MTkzMzAyMzIzNzkyODE0NjcwMjU5NjI5MTU3OTc4NDc2MDg0NTYxNDUwMjIzOTQyMzQxMjExNDU4NjcwNTkwMjg5MTY5NzExNw=="
}